torch/fx/graph.py

```diff
@@ -1089,7 +1136,7 @@ def _target_to_str(self, target : Target) -> str:
         return op
 
     @compatibility(is_backward_compatible=True)
-    def python_code(self, root_module: str) -> PythonCode:
+    def python_code(self, root_module: str, verbose: bool = False) -> PythonCode:
```

Comment:
make it kwarg only. Also you're committing to this kwarg being backwards compatible